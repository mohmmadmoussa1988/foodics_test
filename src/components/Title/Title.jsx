import React from 'react';
import { TitleWrapper } from './Title.styles';

const Title = ({ children }) => {
    return (
        <TitleWrapper>
            {children}
        </TitleWrapper>
    )
}

export default Title;