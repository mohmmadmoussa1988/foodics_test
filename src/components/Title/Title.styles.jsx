import styled from 'styled-components';

export const TitleWrapper = styled.h2`
color:#173753;
font-family: 'SF-bold';
font-size:34px;
line-height:45px;
margin-bottom:40px;
margin-top:0px;
`;
