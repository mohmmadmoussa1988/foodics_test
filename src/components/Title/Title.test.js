import React from "react";
import { configure, shallow } from "enzyme";
import Title from "./Title";
import toJson from "enzyme-to-json";
describe(`<Title />`, () => {
  it("should render Title", () => {
    const wrapper = shallow(<Title />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
