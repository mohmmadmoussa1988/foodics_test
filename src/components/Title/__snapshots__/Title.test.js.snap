// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render Title 1`] = `<styled.h2 />`;
